kvmagent/kvmagent/plugins/hygon_device_plugin.py

```
@@ -0,0 +1,550 @@
+                        "deviceType": "'NTBCCP",
+                        "deviceId": "1468'",
+                        "driverStatus": "drv=hct",
+                        "isMasterPsp": false,
+                        "vendorIdx": null,
+                        "state": "Enabled"
+                    },
+                    ...
+                ]
+            }
+        """
+        cmd = jsonobject.loads(req[http.REQUEST_BODY])
```

Ruff (0.14.7)

Local variable `cmd` is assigned to but never used

Remove assignment to unused variable `cmd`

(F841)